### PR TITLE
Update blast error message for failed submission

### PR DESCRIPTION
**Pull request by imransl**
## Description
Change the blast error message for all jobs failing.

XD: https://xd.adobe.com/view/f585eb32-fbdd-4233-9451-bbb965678c1c-a0d6/screen/0c0b6eed-35b4-4b91-9ccb-de8899af197b?fullscreen

*NOTE*: The changes proposed in the ticket were already available except for blast submission error message. Hence, the one liner for this PR.

## Related JIRA Issue(s)
[ENSWBSITES-1836](https://www.ebi.ac.uk/panda/jira/browse/ENSWBSITES-1836)

## Deployment URL(s)
http://update-blast-error-messages.review.ensembl.org

## Views affected
BLAST

### CODE REVIEW

**Comment by imransl** [comment]
From the discussion with Andrea today, we found out the following changes need to be made:

1. Submission failed (track API): Submission failed
2. Submission failed after all jobs failed (jDispatcher): All jobs have failed individually
3. Some job(s) failed: Results no longer available along with the job failed message for the sequence

**Comment by azangru** [comment]
Here's something we missed in [Jyo's PR](https://github.com/Ensembl/ensembl-client/pull/899)

image

If a single job in a list of jobs grouped by sequence fails, you can see that the name of the species is greyed out (i.e. inactive); but there is still a show/hide chevron next to the species name (which does nothing), and the cursor changes to a pointer when mousing over the grey species name, misleadingly suggesting that you can click on it.

Up to you whether you would like to be fixed in this PR, or in a different one.

File: src/content/app/tools/blast/components/blast-submission-header/BlastSubmissionHeader.tsx [hunk 1]
```diff
@@ -200,6 +204,16 @@ const ControlsSection = (props: {
         </div>
       </div>
     );
+  } else if (haveAllJobsFailed(submission)) {
+    return (
+      <div className={styles.controlsSection}>
+        <DeleteButton onClick={onDelete} disabled={isInDeleteMode} />
+        <div className={styles.errorMessage}>
+          <span>{ALL_JOBS_FAILED_WARNING}</span>
+          <QuestionButton helpText={<FailedSubmissionHelpText />} />
```
Comment (azangru) [inline src/content/app/tools/blast/components/blast-submission-header/BlastSubmissionHeader.tsx:213]: Didn't we say that we are removing the question button? I can't remember.

image

File: src/content/app/tools/blast/utils/blastSubmisionTypeNarrowing.ts [hunk 2]
```diff
@@ -39,3 +39,13 @@ export const isFailedBlastSubmission = (
     return 'error' in submission;
   }
 };
+
+export const haveAllJobsFailed = (
+  submission?: SuccessfulBlastSubmission
+): boolean => {
+  if (!submission) {
```
Comment (azangru) [inline src/content/app/tools/blast/utils/blastSubmisionTypeNarrowing.ts:46]: Just `return submission?.results?.every((job) => job.status === 'FAILURE') ?? false`?

Comment (imransl) [reply src/content/app/tools/blast/utils/blastSubmisionTypeNarrowing.ts]: ✅

**Review by azangru — APPROVED** [review]

**Pull request merged**